### ISSUE TITLE
Latest release on terraform registry is not updated

### ISSUE THREAD

**Issue by santimar** [ansible/terraform-provider-ansible#80]
At the moment, the latest version of this provider on the Terraform registry is `v1.1.0`, which is ~9 months old.

Since then there have been a few updates, most notably #40 and #67 

Is there any reason for such a delay for the new release? 
The last commit on this repo was 3 months ago, it doesn't seem to be getting much attention

**Comment by fbjerggaard** [comment]
Yeah, right now I'm running my own build locally with my fix in it, which is not really a great way to do it in the long run.
It would be great if a new release could be released, and probably automate the process with actions

**Comment by ralgar** [comment]
Wholeheartedly agree with this. It's disappointing to see this project advertised on social platforms, given a stable release right off the bat (v1.1.0 isn't close to production-ready IMO), and then receive no attention at all for months.

**Comment by santimar** [comment]
Good, closing since the ball is rolling again.
